components/Info/DangerModal.tsx

```diff
@@ -17,9 +17,8 @@ const ConfirmationModal = ({ showModal, setDangerModalVisible }: DangerModalProp
   const queryClient = useQueryClient()
 
   async function handleDeleteAllRecords(): Promise<void> {
-    void queryClient.invalidateQueries(['records'])
     await deleteAllRecords()
```

Comment:
Having this below the invalidate query step resulted in a situation where the delete all button would have to be pressed twice to see the result of having deleted all records on the stats screen.